app/helpers/application_helper.rb

```diff
@@ -78,4 +61,18 @@ def gov_link_to_unless_current_user(user:, **options)
   def present(model, presenter_class = nil)
     (presenter_class || [model.class, :Presenter].join.demodulize.constantize).new(model)
   end
+
+  def search_by_caseworker_path(user_id)
+    filter = { assigned_status: user_id, application_status: 'open' }
+    search_application_searches_path(filter:)
+  end
+
+  def link_to_search_by_caseworker(user_name, user_id)
+    govuk_link_to(
+      user_name,
+      search_by_caseworker_path(user_id),
+      data: { turbo: false },
+      no_visited_state: true
```

Comment:
What does this `no_visited_state` imply - it doesn't read clearly to me, first though was that it should be `not_visited_state` but not sure if this is right as I don't know the context.